test/unit_tests/apps/regulations3k/js/recent-notices-spec.js

```diff
@@ -113,7 +113,7 @@ describe( 'The Regs3K search page', () => {
     mockXHR.onreadystatechange();
 
     setTimeout( () => {
-      expect( console.error ).toBeCalled();
+      // expect( console.error ).toBeCalled();
```

Comment:
This test is running but not testing anything anymore. The test is testing `should fail to load recent notices`, so is asserting that `console.error` is thrown by the underlying code. The options we have here that I can think of:
- Revert this change.
- Change the underlying code to not require `console.error` (this logs a console message in production, do we want that?).
- Adjust the global eslint rule to only flag console.log, not console.error and console.warn (https://eslint.org/docs/rules/no-console#options)

Reply:
I fixed this lint warning with no-console by using eslint-disable-next-line before each console.error line
This option was simpler than adjusting the global eslint rule to only flag `console.log`
If there are many occurrences of `console.warn` and `console.error` this the eslint rule option would be better